submitresult.go

```diff
@@ -84,15 +84,14 @@ func buildErrorScanResultModel(stepID string, err error) models.ScanResultModel
 
 func (c *resultClient) uploadErrorResult(stepID string, err error) error {
 	result := buildErrorScanResultModel(stepID, err)
-	return c.uploadResults(result)
-}
-
-func (c *resultClient) uploadResults(result models.ScanResultModel) error {
-	bytes, err := json.MarshalIndent(result, "", "\t")
+	resultBytes, err := json.MarshalIndent(result, "", "\t")
 	if err != nil {
-		return fmt.Errorf("failed to marshal results: %v", err)
+		failf("failed to marshal results: %v", err)
 	}
+	return c.uploadResults(resultBytes)
+}
 
+func (c *resultClient) uploadResults( /*result models.ScanResultModel, */ bytes []byte) error {
```

Comment:
I guess this commented out parameter can be removed.

Reply:
Updated in https://github.com/bitrise-steplib/steps-project-scanner/pull/148/commits/bdcb0dae4a42cf642571062fc889b7db71efd8b4